chia/cmds/data_funcs.py

```diff
@@ -25,39 +25,39 @@ async def get_client(rpc_port: Optional[int]) -> Tuple[DataLayerRpcClient, int]:
 
 async def create_data_store_cmd(rpc_port: Optional[int], table_string: str) -> Optional[Dict[str, Any]]:
     # TODO: nice cli error handling
-
+    client = None
+    response = None
     try:
         client, rpc_port = await get_client(rpc_port)
         response = await client.create_data_store()
     except aiohttp.ClientConnectorError:
         print(f"Connection error. Check if data is running at {rpc_port}")
-        return None
     except Exception as e:
         print(f"Exception from 'data': {e}")
-        return None
-
-    client.close()
-    await client.await_closed()
+    finally:
+        if client is not None:
+            client.close()
+            await client.await_closed()
     return response
 
 
 async def get_value_cmd(rpc_port: Optional[int], store_id: str, key: str) -> Optional[Dict[str, Any]]:
     # TODO: nice cli error handling
-
+    client = None
+    response = None
     store_id_bytes = bytes32(hexstr_to_bytes(store_id))
```

Comment:
I believe we prefer `bytes32.from_hexstr()` over `bytes32(hexstr_to_bytes())` - several instances here